### PR TITLE
fix(spinner): override wrong tokens mapping

### PR DESCRIPTION
**Pull request by mivaylo**
## PR Checklist

Please check if your PR fulfills the following requirements:

- [x] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)
- [ ] If applicable, have a visual design approval

## PR Type

What kind of change does this PR introduce?

- [x] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [ ] Documentation content changes
- [ ] Other... Please describe:

## What is the current behavior?

image

Issue Number: CDE-1513

## What is the new behavior?

image

## Does this PR introduce a breaking change?

- [ ] Yes
- [x] No

## Other information

Spinner inverse tokens were correct but their mapping to the global ones is wrong. This overrides the wrong mapping and these tokens are not used in other places so we are safe with the fix.

Update: after discussion with @valentin-mladenov and design team we've found out that two more tokens have incorrect mapping, so we are adding them too in this PR.

Another improvement that the PR includes is providing a more meaningful demo for the inverse spinner (with proper background) in the demo app and the storybook

### CODE REVIEW

**Comment by valentin-mladenov** [comment]
Follow up of #1488

**Review by valentin-mladenov — CHANGES_REQUESTED** [review]
Check my comments

File: .storybook/stories/spinner/spinner.stories.ts [hunk 1]
```diff
@@ -35,7 +35,14 @@ export default {
...
   template: `
-    <div style="text-align: center">
+    <style>
+      .spinner-inverse-container {
+        background: var(--cds-alias-object-container-background-inverse);
+        color: var(--cds-alias-typography-color-100);
+        padding: 20px;
+      }
+    </style>
+    <div style="text-align: center" [ngClass]="{ 'spinner-inverse-container': clrInverse }">
```
Comment (valentin-mladenov) [inline .storybook/stories/spinner/spinner.stories.ts:45]: Wouldn't it be better to use like the following.

```suggestion
<div style="text-align: center" [class.spinner-inverse-container]="clrInverse">
```

**Review by valentin-mladenov — APPROVED** [review]
LGTM. Check my comment.

File: .storybook/stories/spinner/spinner.stories.ts (hunk 1 above)
Comment (mivaylo) [reply .storybook/stories/spinner/spinner.stories.ts]: thanks, updated

File: .storybook/stories/spinner/spinner.stories.ts [hunk 2]
```diff
@@ -46,12 +46,25 @@ const SpinnerTemplate: StoryFn = args => ({
   props: args,
 });
 
+const SpinnerInverseTemplate: StoryFn = args => ({
+  template: `
+    <div style="text-align: center; background: var(--cds-alias-object-container-background-inverse)">
```
[the remark this answers is not in the document]
Comment (mivaylo) [reply .storybook/stories/spinner/spinner.stories.ts]: fixed : https://github.com/vmware-clarity/ng-clarity/pull/1500/commits/324924c9f6fded9f2258a3d138383fd1d46fb6e8

File: projects/ui/src/temp-overrides.scss [hunk 3]
```diff
@@ -12,6 +12,10 @@
   --cds-alias-object-interaction-background-selected: #{tokens.$cds-global-color-blue-75};
   --cds-alias-object-interaction-background-selected-hover: #{tokens.$cds-global-color-construction-100};
   --cds-alias-object-interaction-background-selected-active: #{tokens.$cds-global-color-construction-200};
+  --cds-alias-object-container-background-inverse: #{tokens.$cds-global-color-construction-600};
+  --cds-alias-object-container-background-inverse-tint: #{tokens.$cds-global-color-construction-800};
+  --cds-alias-object-container-background-inverse-shade: #{tokens.$cds-global-color-construction-500};
+  --cds-alias-object-container-background-dark: #{tokens.$cds-global-color-construction-200};
```
[the remark this answers is not in the document]
Comment (mivaylo) [reply projects/ui/src/temp-overrides.scss]: fixed: https://github.com/vmware-clarity/ng-clarity/pull/1500/commits/81fb054985d757606c7eb90357d6d18da96b7622

**Pull request merged**